src/EffectTransform.js

```diff
@@ -123,6 +123,10 @@ class EffectTransform {
      * @returns {Uint8ClampedArray} dst filled with the transformed color
      */
     static transformColor (drawable, color4b, dst, effectMask) {
+        const effects = drawable.getEnabledEffects() & effectMask;
+        // If no effects are enabled, don't get anything, create anything, or set anything; just give back the input.
+        if (effects === 0) return color4b;
```

Comment:
I like this early return. This will definitely help by skipping the other if statements that would in this case all return false and perform no further work.

However we must move this early exit to after `dst` is initialized and return `dst`. That will match the documentation and the current behaviour.